src/hooks/useInfiniteCarousel.ts

```
@@ -0,0 +1,148 @@
+
+  return {
+    slides,
+    currentIndex: hasSlides ? currentIndex : 0,
+    isAnimate: hasSlides ? isAnimate : false,
+    dragOffset,
+    displayIndex,
+    totalOriginalSlides,
+    isSwiped: hasSlides && Math.abs(currentX - startX) > 5,
+    handlers: hasSlides
+      ? {
+          onMouseDown: (e: React.MouseEvent) => handleDragStart(e.clientX),
```

ESLint

[error] Expected indentation of 8 spaces but found 10.

(indent)